vtex-apps/minicart
# Add itemCountMode prop

#### What is the purpose of this pull request?

~Display the count of all items in the cart with quantities (showTotalItemsQty from v1).~

Add a new `itemCountMode` prop to the `minicart.v2` block.

This prop controls how the quantity badge shown in the minicart icon counts the number of items.

#### What problem is this solving?

~We have customers asking for this behavior.~

Since this prop controls how the quantity badge counts items, we enable customers to display the quantity items taking into account the quantity of each item, not only distinct items.

#### How should this be manually tested?

~Change the default value to true, add multiple quantities of products and cart and observe the cart badge displaying the count.~

Go to this [Workspace](https://dux--coretech.myvtex.com/) and add any item to the cart, with a quantity greater than one.

#### Types of changes
